sagemaker-pipelines/step-decorator/computer-vision-examples/computer-vision-pipeline.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.        tuner=tuner,
</pre><p> </p><p>This is an obsoleted TuningStep interface, that passing the tuner instance and inputs directly to the step.</p><p>We've seen a lot of bugs relating to such usage so we introduced a new job step interface to improve, which is firstly passing PipelineSession instance to the estimator like you've already done</p><p> </p><p>Then, generate the <code>step_args</code> via tuner.fit . The PipelineSession would delay the creation of training job and get the job request arguments instead. Finally, pass the <code>step_args</code> to the TuningStep</p><p> </p><pre class="ql-syntax" spellcheck="false">step_args = tuner.fit(
    inputs=inputs={"train": train_channel, "validation": validation_channel},
)

step = TuningStep(
    name="MyTuningStep",
    step_args=step_args,
)
</pre><p> </p><p>More details on the new job step interface can be found in <a href="https://sagemaker.readthedocs.io/en/stable/amazon_sagemaker_model_building_pipeline.html#pipeline-session" target="_blank">https://sagemaker.readthedocs.io/en/stable/amazon_sagemaker_model_building_pipeline.html#pipeline-session</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Thanks! I was not aware that the step interface had changed. I like the new interface more so will definitely update this.